TASVideos/Pages/Shared/Components/UserMovies/Default.cshtml

```diff
@@ -11,7 +11,7 @@
 		@foreach (var item in Model)
 		{
 			<tr>
-				<td><timezone-convert asp-for="@item.UploadTimestamp" date-only="true" /></td>
+				<td><timezone-convert asp-for="@item.UploadTimestamp" date-only="true" relative-time="true" /></td>
```

Comment:
It seems you want to use relative time pretty often, maybe true should be the default and optional?

Reply:
Yeah, I didn't realize there were so many separate places where it's useful to have.